recipe/meta.yaml

```diff
@@ -13,13 +13,16 @@ source:
 
 build:
   number: 0
-  skip: true
```

Comment:
Can you make this `skip: true   # [py>35]` to skip python3.6 as not all your dependencies are on python3.6 yet?